src/components/PictureCard.js

```diff
@@ -4,10 +4,9 @@ import { Grid } from "@material-ui/core";
 
 const CardItem = (props) => {
   return (
-    <Grid item xs={12} lg={4}  >
-      <div align='center' style={{ height: '50%' }}>
-        <img src={props.src} alt={props.alt} style={{ width: '300px', height: '220px' }} /> <p>{props.children}</p>
-      </div>
+    <Grid item xs={12} lg={4} style={{margin:'auto'}} align='center' >
+        <img src={props.src} alt={props.alt} />
+         <p style={{padding:'0 30px 0 30px'}}>{props.children}</p> 
```

Comment:
Make sure the img and p tags are properly indented. Also can remove the last 2 values from padding property.
```suggestion
    <Grid item xs={12} lg={4} style={{margin:'auto'}} align='center' >
        <img src={props.src} alt={props.alt} />
        <p style={{padding:'0 30px'}}>{props.children}</p> 
```